spec/factories/users.rb

```diff
@@ -1,7 +1,12 @@
 FactoryGirl.define do
+  factory :role do
+    sequence(:title) { |n| "role#{n}" }
+  end
+
   factory :user do
     sequence(:name) { |n| "User#{n}" }
     sequence(:email) { |n| "user#{n}@test.com" }
     sequence(:password) { |n| "123456#{n}" }
+    role
```

Comment:
user returns role?))

Reply:
https://stackoverflow.com/questions/11530616/how-to-create-belongs-to-relationship-in-factory-girl-sequence is it wrong?

Reply:
It's better to put the role as a parameter while invoking the factory in tests since you will have determined roles.